lib/CRUD/sql_screen.dart

```diff
@@ -0,0 +1,165 @@
+import 'package:apism/CRUD/sql_helper.dart';
+import 'package:flutter/material.dart';
+
+class SqlHomeScreen extends StatefulWidget {
+  const SqlHomeScreen({super.key});
+
+  @override
+  State<SqlHomeScreen> createState() => _SqlHomeScreenState();
+}
+
+class _SqlHomeScreenState extends State<SqlHomeScreen> {
+  List<Map<String, dynamic>> _users = [];
+
+  bool _isLoading = true;
+  // This function is used to fetch all data from the database
+  void _refreshusers() async {
+    final data = await SQLHelper.getItems();
+    setState(() {
+      _users = data;
+      _isLoading = false;
+    });
+  }
+
+  @override
+  void initState() {
+    super.initState();
+    _refreshusers();
+  }
+
+  final TextEditingController _titleController = TextEditingController();
+  final TextEditingController _descriptionController = TextEditingController();
+
+  
+  void _showForm(int? id) async {
+    if (id != null) {
+      // id == null -> create new item
+      // id != null -> update an existing item
+      final existingUserList =
+      _users.firstWhere((element) => element['id'] == id);
+      _titleController.text = existingUserList['title'];
+      _descriptionController.text = existingUserList['description'];
+    }
+
+    showModalBottomSheet(
+        context: context,
+        elevation: 5,
+        isScrollControlled: true,
+        builder: (_) => Container(
```

Comment:
This widget created with container should be create as a separte widget class if it will be reused

Reply:
Done Sir